pokemongo_bot/cell_workers/catch_lured_pokemon.py

```diff
@@ -12,39 +13,52 @@ class CatchLuredPokemon(BaseTask):
     def work(self):
         lured_pokemon = self.get_lured_pokemon()
         if lured_pokemon:
-            self.catch_pokemon(lured_pokemon)
+            for pokemon in lured_pokemon:
+                self.catch_pokemon(pokemon)
 
     def get_lured_pokemon(self):
+        forts_in_range = []
+        pokemon_to_catch = []
         forts = self.bot.get_forts(order_by_distance=True)
 
         if len(forts) == 0:
             return False
 
-        fort = forts[0]
-        details = fort_details(self.bot, fort_id=fort['id'],
```

Comment:
@TheSavior yes... we will a size-limited dict for forts with more data and not only the timer.